src/action-creators/alert.ts

```diff
@@ -5,9 +5,10 @@ interface Options {
   alertType?: string
   showCloseLink?: boolean
   clearDelay?: number
+  isInline?: boolean
 }
 
-let clearAlertTimeoutId: number | null = null // eslint-disable-line fp/no-let
+let clearAlertTimeoutId: ReturnType<typeof setTimeout> | null = null // eslint-disable-line fp/no-let
```

Comment:
Number type can't be declared for the return type of setTimeout hence replaced it with `ReturnType<typeof setTimeout>`.

Reply:
That seems reasonable. Thanks.